attributes/default.rb

```diff
@@ -38,6 +38,14 @@
 # Collect EC2 tags, set to 'yes' to collect
 default['datadog']['collect_ec2_tags'] = nil
 
+# A regex used to blacklist the tags that the host sends to datadog.
```

Comment:
maybe we could reword this to: `Set this regex to exclude some Chef node tags from the host tags that the datadog handler sends to Datadog`

We could then remove the next line (`This affects not only event tags but also the host-level tags.`)

Let me know if that sounds better to you or not